docusaurus.config.js

```diff
@@ -133,6 +133,10 @@ const config = {
 
     navbar: {
       hideOnScroll: true,
+      logo: {
+        alt: 'KNDP Logo',
+        src: '/img/logo.png',
```

Comment:
@BurlacuLucian No this logo, from GitHub repository: https://avatars.githubusercontent.com/u/141832625?s=200&v=4